app/schema.json

```diff
@@ -36,6 +36,16 @@
     "chromInfoPath": { "type": "string" }
   },
   "definitions": {
+    "data": {
+      "type": "object",
+      "properties": {
+        "type": { "type": "string" },
+        "tilesetInfo": { "type": "object" },
+        "children": { "type": "array" },
```

Comment:
Just curious what this children field is used for. I see that it was part of the existing data object properties.

Reply:
It can be used to create tracks that display the ratio of two datasets. I just added a section to the docs explaining this: https://github.com/higlass/higlass/pull/871/files#diff-74d725479bc068778bbd4ad8e310a67dR352-R372

Reply:
Ok thanks!